aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -159,6 +159,11 @@ def validate_pseudos(self):
         pseudos = self.ctx.inputs.get('pseudos', None)
         pseudo_family = self.inputs.get('pseudo_family', None)
 
+        if pseudo_family:
+            import warnings
+            from aiida.common.warnings import AiidaDeprecationWarning
```

Comment:
Similar to our discussion in #569, shouldn't we create a validator instead of raising a warning, which will just wind up in the daemon logs?

Reply:
very good point